docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/pull-request-scan.adoc

```diff
@@ -0,0 +1,54 @@
+=== Pull Request Scans
+
+Pull request scans are event driven scans that are run on branches with open Pull Requests (PR) from your integrated repositories. The scan results are based on the default *Enforcement* thresholds or custom configurations.
+
+[.task]
+
+=== Configure Pull Request Scans
+
+Configure PR scan settings using the *Comment Bot* Enforcement threshold.
+
+[.procedure]
+
+. In *Application Security* select *Projects* > *Action* menu.
+. Select *Enforcement* from the menu.
++
+image::application-security/enfor-1.1.png[]
+
+. In a code category, drag the *Comments bot* threshold rule arrow to your required severity.
++
+image::application-security/pr-sca-enforcement.png[]
+
+. Select *Save*
+
+For more information on Enforcement, refer to xref:enforcement.adoc[Enforcement].
+
+=== Manage Pull Request Scan Results
+
+You can manage pull request scan results within the pull request itself or through the Prisma Cloud Console.
+
+==== On the Pull Request:
+
+Errors detected by a PR scan triggered by changes to a resource are displayed as a PR comment within the pull request. Details of the error and suggested remediation can be viewed by clicking the arrow in the comment.
```

Comment:
We should note that SCA PR comments are for any vulnerability in a modified package manager file. Secrets PR comments are only for new or modified lines of code that include a secret finding.

You cover IaC well.